.eslintrc.json

```diff
@@ -1,5 +1,12 @@
 {
-    "extends": "eslint:recommended",
+    "parser": "typescript-eslint-parser",
```

Comment:
@danielbeardsley is this supposed to be `@typescript-eslint/parser`? I am getting an error saying it can't find this module but if we use the suggestion I added then it works.

```suggestion
    "parser": "@typescript-eslint/parser",
```
![eslint not found local build](https://user-images.githubusercontent.com/22064420/140393285-53e518e5-827b-465d-84e7-250425805ae1.png)

Reply:
Fixed! Thank you. This lead me to realize the eslint setup wasn't being used like I thought it was.